src/gvpn_controller.py

```diff
@@ -110,7 +117,7 @@ def create_connection_req(self, data):
         self.ondemand_create_connection(uid, send_req=True)
 
     def serve(self):
-        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"])
+        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"]) # Select sockets ready to be served
```

Comment:
[This is a good explanation of how `select` works](http://beej.us/guide/bgnet/output/html/multipage/advanced.html#select).